### PR TITLE
Delete validate-mainnet package

### PR DESCRIPTION
**Pull request by erikd**
The functionality of this package was included in cardano-node.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,33 +1,8 @@
...
-**/latex/*.out
-**/latex/*.toc
-**/latex/*.fmt
-**/latex/*.fot
-**/latex/*.cb
-**/latex/*.cb2
-**/latex/*.lb
-**/latex/*.pdf
-**/latex/*.fdb_latexmk
-**/latex/auto
```
Comment (Jimbo4350) [inline .gitignore:16]: What were these doing here?

File: cabal.project [hunk 2]
```diff
@@ -1,7 +1,6 @@
 packages:
   cardano-ledger     cardano-ledger/test
   crypto crypto/test
-  validate-mainnet
```
Comment (Jimbo4350) [inline cabal.project:4]: :+1:

**Review by Jimbo4350 — APPROVED** [review]
LGTM

**Review by CodiePP — APPROVED** [review]
lgtm. if the ledger team thinks this is not necessary anymore.

**Pull request merged**